Task_1/Task_1/src/components/to_do_list/List.jsx

```diff
@@ -0,0 +1,23 @@
+import TaskBar from "./task_bar";
+
+const List = (props) => {
+  if (props.list.length == 0) {
```

Comment:
it can be written as !props.list.length. short hand and more optimized syntax

Reply:
Done